DataCapturing/DataCapturingService.swift

```diff
@@ -161,8 +167,8 @@ public class DataCapturingService: NSObject {
      running.
 
      - Throws:
-        - `DataCapturingError.isPaused` if the service was paused and thus stopping it makes no sense.
-        - `DataCapturingError.noCurrentMeasurement` If there is no measurement to stop.
+        - `PersistenceError` If the currently captured measurement was not found in the database.
+        - Some unspecified errors from within *CoreData*.
```

Comment:
documentation was wrong here. These are the true exceptions thrown at this place.